recipes/newrelic/apm/php/debian.yml

```diff
@@ -372,21 +440,8 @@ install:
         - |
           echo -e "{{.ARROW}}Send queryable transactions{{.GRAY}}"
           cd {{.TMP_INSTALL_DIR}}
-          NR_INSTALL_LOG={{.TMP_INSTALL_DIR}}/nrinstall.log
-          WEB_PHP_INI_DIR=$(sed -n 's/.*final pi_inidir_dso=\(.*\)\/conf.d/\1/p' $NR_INSTALL_LOG)
-          WEB_NR_INI_DIR=$(sed -n 's/.*final pi_inidir_dso=\(.*\/\)/\1/p' $NR_INSTALL_LOG)
-          if [ -z "${WEB_PHP_INI_DIR}" ]; then
-            WEB_PHP_INI_DIR=$WEB_NR_INI_DIR
-          fi
-          # Fallback to cli ini dir if there is no web specific ini dir
-          if [ -z "${WEB_PHP_INI_DIR}" ]; then
-            WEB_PHP_INI_DIR=($(sed -n 's/.*final pi_inidir_cli=\(.*\)\/conf.d/\1/p' $NR_INSTALL_LOG))
-            WEB_NR_INI_DIR=($(sed -n 's/.*final pi_inidir_cli=\(.*\/\)/\1/p' $NR_INSTALL_LOG))
-            if [ -z "${WEB_PHP_INI_DIR}" ]; then
-              WEB_PHP_INI_DIR=$WEB_NR_INI_DIR
-            fi
-          fi
 
+          sudo -u $SUDO_USER sudo chmod 666 /var/log/newrelic/*
```

Comment:
This makes me uncomfortable. A malicious user on the system could use this to fill the log volume before performing an attack they don't want to be logged in the system's logs.

Alternatives to consider are (in no particular order):
1. Configure newrelic.ini to use a different filename to avoid the need for a world writable file.
2. `chown` the file to the primary user we expect to be writing to the log file (potentially emitting a warning about the others being unable to write to it)
3. Detect the incompatibility and fail the installation with a unique error code. If this ends up being common, address it then.
4. Double check with the security team and ship this as-is, if they are okay with the risk.

Reply:
Those are valid concerns.  The change here is only to allow php-cli to send debug msg to the logs when web services are installed; in the case of php-cli only installation, it is not an issue, so not a reason to fail the recipe. 

Some options:
1) If we installed for web, make a note that for the php-cli to send log messages the user would need to change the permissions.  There would be no need for us to change any permissions as all the web users are able to write to the file.
2) investigate behavior of chowning to sudo_user.
3) Double check with the security team and ship this as-is, if they are okay with the risk.

Reply:
I like option one. I'd also be okay with not logging that message until this scenario proves to be problematic. Logging a message for a scenario that ends up not being an issue in practice would result in more confusion than help.

Reply:
I'm okay also not logging the message.
I investigate #2, but I'll just remove the chmod now.